lua/nvim-tree/explorer/init.lua

```diff
@@ -238,17 +241,17 @@ function Explorer:refresh_parent_nodes_for_path(path)
   -- collect parent nodes from the top down
   local parent_nodes = {}
   NodeIterator.builder({ self })
-    :recursor(function(node)
-      return node.nodes
-    end)
-    :applier(function(node)
-      local abs_contains = node.absolute_path and path:find(node.absolute_path, 1, true) == 1
-      local link_contains = node.link_to and path:find(node.link_to, 1, true) == 1
-      if abs_contains or link_contains then
-        table.insert(parent_nodes, node)
-      end
-    end)
-    :iterate()
+      :recursor(function(node)
+        return node.nodes
+      end)
+      :applier(function(node)
+        local abs_contains = node.absolute_path and path:find(node.absolute_path, 1, true) == 1
+        local link_contains = node.link_to and path:find(node.link_to, 1, true) == 1
+        if abs_contains or link_contains then
+          table.insert(parent_nodes, node)
+        end
+      end)
+      :iterate()
```

Comment:
Unnecessary whitespace change.

You can `make style-fix`

Reply:
Probably a result of using `:Format`. I'll use the make style next time.